### ISSUE TITLE
styleOfPermalinks is not a function

### ISSUE THREAD

**Issue by AleksandrHovhannisyan** [valeriangalliat/markdown-it-anchor#96]
The documentation here:

https://github.com/valeriangalliat/markdown-it-anchor/#permalinks

Says that you can do this:

```js
const anchor = require('markdown-it-anchor')
const md = require('markdown-it')()

md.use(anchor, {
  permalink: anchor.permalink.styleOfPermalink(permalinkOpts)
})
```

But this throws an error. Is `styleOfPermalink` not an actual method we can invoke? If not, what is the correct syntax?

Thanks!

**Comment by valeriangalliat** [comment]
Ah yeah I meant `styleOfPermalink` as a placeholder for any of the permalink methods documented below, but that's not necessarily obvious.

I modified the example to have `anchor.permalink[styleOfPermalink](permalinkOpts)` to make it more clear that it's a variable, and I added a paragraph below saying it needs to be one of the styles documented afterwards.

Let me know if that heps! Cheers :)
